Router.svelte

```diff
@@ -491,6 +510,30 @@ let componentObj = null
 // Listen to changes in the $loc store and update the page
 // Do not use the $: syntax because it gets triggered by too many things
 const unsubscribeLoc = loc.subscribe(async (newLoc) => {
+    if (window.location.hash.match(/^#[^\/]\w*/)) {
+        if (!lastLoc) {
+            // Could be another behavior, but this one seems right to me
+            window.location.hash = '#'
```

Comment:
As explained, this creates a new entry in the history stack, so it will break back navigation.